lib/DDG/Spice/IndegoBikeshareLocations.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::IndegoBikeshareLocations;
```

Comment:
@moollaza great! That being said, once I create the new files, should I close this pull request and submit a new one? Will you see it if I do and continue where we left off?